package.json

```diff
@@ -2,7 +2,7 @@
   "name": "random-theme-switcher",
   "displayName": "Random Theme Switcher",
   "description": "A simple extension that randomly choose a theme",
-  "version": "0.1.3",
+  "version": "1.0.0",
```

Comment:
non farei la versione 1, preferirei fare la versione `0.2`

Reply:
Sistemato 👍🏻

Anch'io tengo i valori delle versioni bassi, 
non so perchè questa volta non l'abbia fatto, 
ma ok.

Idea per l'icona: si potrebbero aggiungere dei "?" (punti interrogativi) ruotati sulle facciate del cubo, magari con tinte diverse, per rendere ancor di più la casualità dei temi.

Reply:
creare un nuovo logo non sarebbe male, non sono bravo e l'ho creato in poco tempo.